README.md

```diff
@@ -1,37 +1,34 @@
 # container-image-template
 
-[![Build Status](https://travis-ci.org/container-images/container-image-template.svg?branch=master)](https://travis-ci.org/container-images/container-image-template)
+[![Build Status](https://travis-ci.org/container-images/systemtap.svg?branch=master)](https://travis-ci.org/container-images/systemtap)
 
-This is a template of a git repository which is a source for building container images.
+SystemTap running in a container.
 
 
-There are two templates within this repository on two different branches:
+## Usage
 
- 1. `master` — template for images with multiple versions
+You can use atomic command to run systemtap container:
 
- 2. `single` — image with only one single version
+```
+$ atomic run modularitycontainers/systemtap
+```
 
+Once you are in the shell, you can try some simple systemtap scripts:
 
-## Description
+1. "hello world" with systemtap:
+  ```
+  $ stap -e 'probe begin { println("hello world"); }'
+  hello world
+  ^C
+  ```
 
- * `help` - a directory containing `help.md` file. This serves as a base for container help page. It can be templated on per-version and per-os\_variant bases. On executing `make doc` in a `<version>/<os_variant>/` directory, it will search for `<version>/root/help.values` and `<version>/<os_variant>/root/help.values`, source these two scripts (if they exist) and run `envsubst` (environment variable substitution) to obtain `help.md` specific to the currently `make`-ed image.
- * `LICENSE` — pick the right license
- * `Makefile` — so it's easy to build, run, test
- * `openshift-template.yml` - general OpenShift template. A postfix template https://github.com/container-images/postfix/blob/master/openshift-template.yml
-  * update it. It includes several commands what to update
- * `README.md` — global documentation for the whole service
-  * what is it, how to build, how to use
- * `tests/` — these tests should verify that the image works
+2. script to print backtrace from a selected kernel function:
```

Comment:
Sorry for digging. This is an example?
Does it make to include some real example like is mentioned there https://sourceware.org/systemtap/examples/?

Do you plan to create a test wether systemtap works properly?

Reply:
Good point with the example, I'll write down that those examples you are linking are actually available inside.

Regarding tests: I tried, but it was really hard -- the biggest issue is that you need those huge 500MB packages installed in the right version, so I'll write the tests at some point.